<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="open_img">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>590</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开图像</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="text_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;识别结果：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ans">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
         <underline>false</underline>
         <stylestrategy>PreferAntialias</stylestrategy>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Wang Hongshuo</string>
    </property>
   </widget>
   <widget class="ImageWidget" name="show_input_img" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageWidget" name="show_binary_img" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="ImageWidget" name="show_optimized_img" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>600</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="c5">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c6">
    <property name="geometry">
     <rect>
      <x>529</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c7">
    <property name="geometry">
     <rect>
      <x>616</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c2">
    <property name="geometry">
     <rect>
      <x>181</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c1">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c3">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="c4">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>491</y>
      <width>48</width>
      <height>84</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>48</width>
      <height>84</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="text_0">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>41</y>
      <width>64</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;原始图像&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>179</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;二值化&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>318</y>
      <width>48</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;优化后&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>open_img</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>show_input_img</zorder>
   <zorder>show_binary_img</zorder>
   <zorder>show_optimized_img</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>open_img</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
